packages/@webex/plugin-meetings/src/reachability/util.ts

```diff
@@ -43,3 +43,25 @@ export function convertStunUrlToTurnTls(stunUrl: string) {
 
   return url.toString();
 }
+
+/**
+ * Checks if the given server address is a domain name.
+ *
+ * @param {string} url - The server IP or domain name to check.
+ * @returns {boolean} true if the server IP is a domain name, false otherwise.
+ */
+export function isDomainName(url: string): boolean {
```

Comment:
maybe would be worth using an existing library that does this sort of stuff (like ip-regex or is-ip, or maybe there is even something like this already in SDK) instead of inventing our own regex